Engine/EditorUI/Panel/PanelPopups.h

```diff
@@ -22,9 +22,9 @@ class PanelPopups : public Panel
 	void RenderAssetsLoadingPopup();
 
 public:
-	PanelPopupMaterialSelector material_selector_popup;
-	PanelPopupMeshSelector mesh_selector_popup;
-	PanelPopupSceneManagement scene_management_popup;
+	PanelPopupResourceSelector resource_selector_popup;
+	PanelPopupSceneLoader scene_loader_popup;
+	PanelPopupSceneSaver scene_saver_popup;
```

Comment:
why those are not pointers? and why aren't they being initialized?

Reply:
Because the only class using this is the `PanelPopups`, it almost make no difference declaring them as pointers. The only difference will be the memory acces time, because they are allocated in memory directions assigned to the object `PanelPopups`. 

Also, th default constructor of this classes will be used because no initiliazier is specified. Which in this case, it's ok

Reply:
Good to know!!